acceptance/framework/consul/cli_cluster.go

```diff
@@ -3,14 +3,14 @@ package consul
 import (
```

Comment:
Update to the CLI Cluster to use the above code and prevent unnecessary re-compilation.